glue_vispy_viewers/__init__.py

```diff
@@ -1 +1,3 @@
-from .version import __version__
\ No newline at end of file
+from .version import __version__
+BACKGROUND_COLOR = 'black'
+GRIDBOX_COLOR = 'white'
```

Comment:
I would just call this FOREGROUND_COLOR since it could apply to axes, labels, etc.